cookbooks/aws-parallelcluster-install/attributes/default_centos7.rb

```diff
@@ -23,3 +23,22 @@
 default['cluster']['kernel_devel_pkg']['version'] = node['kernel']['release']
 
 default['cluster']['chrony']['conf'] = "/etc/chrony.conf"
+
+# Arm Performance Library
+default['cluster']['armpl']['major_minor_version'] = '21.0'
```

Comment:
Instead of duplicating all this code in all the files let's create instead an `attributes/arm_pl.rb` that contains the common part, we'll keep in the OS specific file only the OS dependent properties. This will simplify a lot the changes in case of update of the library.